content/en/docs/best-practices/common.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to distribute solutions with Operator OLM
+---
+
+## Overview
+
+Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices].
+
+## Provide what are the k8s versions supported by your project
+
+In this CSV manifest, you will check that it provides the spec field `minKubeVersion`. Then, would be recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available which not necessary is the case.
```

Comment:
I'm a little confused here. It looks like we're presenting a CSV manifest based on this description, but I don't see any reference (e.g. link to an existing CSV manifest that provides an example). It's possible I'm missing something obvious?

Reply:
added. Let me know what do you think now